Formula/teleport.rb

```diff
@@ -0,0 +1,47 @@
+class Teleport < Formula
+  desc "Modern SSH server for teams managing distributed infrastructure"
+  homepage "https://gravitational.com/teleport"
+  url "https://github.com/gravitational/teleport/archive/v1.3.2.tar.gz"
+  sha256 "2f2f5aa019c99d1a829f16745aa6d81778945ca29b3e936cc9117aac832e12e1"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath / "src/github.com/gravitational/teleport").install buildpath.children
+    cd "src/github.com/gravitational/teleport" do
+      system "godep", "restore"
+      system "go", "build", "-o", bin / "tctl", "-ldflags", "-w -s", "./tool/tctl"
+      system "go", "build", "-o", bin / "teleport", "-ldflags", "-w -s", "./tool/teleport"
+      system "go", "build", "-o", bin / "tsh", "-ldflags", "-w -s", "./tool/tsh"
+      prefix.install_metafiles
+      cd "web/dist" do
+        system "zip", "-q", "-r", "webassets.zip", "."
+        system "cat webassets.zip >> #{bin}/teleport"
```

Comment:
It's a parallelization bug, which should be reported to them. You can add `ENV.deparallelize` to the begin of the install method, and it will fix that.